android/java/org/chromium/chrome/browser/crypto_wallet/activities/AccountDetailActivity.java

```diff
@@ -54,13 +57,15 @@
 import java.util.concurrent.Executors;
 
 public class AccountDetailActivity
-        extends BraveWalletBaseActivity implements OnWalletListItemClick {
+        extends BraveWalletBaseActivity implements OnWalletListItemClick, ApprovedTxObserver {
     private String mAddress;
     private String mName;
     private boolean mIsImported;
     private TextView mAccountText;
     private ExecutorService mExecutor;
     private Handler mHandler;
+    private AccountInfo[] accountInfos;
```

Comment:
what is the point to have it as a class member?

Reply:
let me remove it, it's only required in Asset details activity.